Formula/itk.rb

```diff
@@ -41,30 +42,32 @@ class Itk < Formula
   fails_with gcc: "5"
 
   def install
-    args = std_cmake_args + %W[
+    # Avoid CMake trying to find GoogleTest even though tests are disabled
+    (buildpath/"Modules/ThirdParty/GoogleTest").rmtree
+
+    args = %W[
       -DBUILD_SHARED_LIBS=ON
-      -DBUILD_TESTING=OFF
       -DCMAKE_INSTALL_RPATH:STRING=#{lib}
       -DCMAKE_INSTALL_NAME_DIR:STRING=#{lib}
-      -DITK_USE_64BITS_IDS=ON
-      -DITK_USE_STRICT_CONCEPT_CHECKING=ON
-      -DITK_USE_SYSTEM_ZLIB=ON
-      -DITK_USE_SYSTEM_EXPAT=ON
-      -DModule_SCIFIO=ON
       -DITKV3_COMPATIBILITY:BOOL=OFF
-      -DITK_USE_SYSTEM_FFTW=ON
+      -DITK_LEGACY_REMOVE=ON
+      -DITK_USE_64BITS_IDS=ON
       -DITK_USE_FFTWF=ON
       -DITK_USE_FFTWD=ON
+      -DITK_USE_SYSTEM_FFTW=ON
       -DITK_USE_SYSTEM_HDF5=ON
       -DITK_USE_SYSTEM_JPEG=ON
       -DITK_USE_SYSTEM_PNG=ON
       -DITK_USE_SYSTEM_TIFF=ON
       -DITK_USE_SYSTEM_GDCM=ON
-      -DITK_LEGACY_REMOVE=ON
+      -DITK_USE_SYSTEM_ZLIB=ON
+      -DITK_USE_SYSTEM_EXPAT=ON
+      -DITK_USE_SYSTEM_DOUBLECONVERSION=ON
+      -DITK_USE_SYSTEM_LIBRARIES=ON
```

Comment:
Enabled general `ITK_USE_SYSTEM_LIBRARIES`. Also added `ITK_USE_SYSTEM_DOUBLECONVERSION`, which isn't enabled by general option and needs to be manually set.

Note that there is a bundled copy of `openjpeg` that cannot be dropped due to https://github.com/InsightSoftwareConsortium/ITK/issues/1807